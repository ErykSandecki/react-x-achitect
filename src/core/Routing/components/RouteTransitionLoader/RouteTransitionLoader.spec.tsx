import { render } from '@testing-library/react';

// components
import RouteTransitionLoader from './RouteTransitionLoader';

describe('RouteTransitionLoader snapshots', () => {
  it('should render RouteTransitionLoader', () => {
    // before
    const { asFragment } = render(<RouteTransitionLoader />);

    // result
    expect(asFragment()).toMatchSnapshot();
  });
});
