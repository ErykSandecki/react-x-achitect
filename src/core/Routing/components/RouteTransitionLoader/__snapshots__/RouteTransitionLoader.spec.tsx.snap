// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RouteTransitionLoader snapshots should render RouteTransitionLoader 1`] = `
<DocumentFragment>
  <div>
    RouteTransitionLoader
  </div>
</DocumentFragment>
`;
